task/scorecard/0.1/scorecard.yaml

```
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: scorecard
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: scorecard
+    tekton.dev/displayName: OSSF scorecard
+
+spec:
+  description: >-
+    This task can be used to calculate the scorecard results for a repository
+
+  params:
+    - name: REPO
+      description: The repository URL to inspect
+      type: string
+    - name: GITHUB-OAUTH_SECRET
+      description: The GitHub secret with credentials
+      type: string
+      default: "github"
+  steps:
+    - name: scorecard
+      image: gcr.io/openssf/scorecard:latest+sha256:c16e433f68799fe69dc0f9675c7ba5d2553846b57262550c86c71165269a7961
```

my bad :man_facepalming: 
```suggestion
      image: gcr.io/openssf/scorecard@sha256:c16e433f68799fe69dc0f9675c7ba5d2553846b57262550c86c71165269a7961
```

---

@dlorenc is `sha256:c16e433f68799fe69dc0f9675c7ba5d2553846b57262550c86c71165269a7961` pointing to `:latest` ? If yes, the digest might become unreliable next time something is pushed on `:latest` and gcr does a garbage collection.

---

All set!